aptos-move/framework/cached-packages/src/aptos_stdlib.rs

```diff
@@ -15,3 +15,16 @@ pub fn aptos_coin_transfer(to: AccountAddress, amount: u64) -> TransactionPayloa
         amount,
     )
 }
+
+pub fn encode_create_resource_account(
+    seed: &str,
+    authentication_key: Option<AccountAddress>,
+) -> TransactionPayload {
+    let seed: Vec<u8> = bcs::to_bytes(seed).unwrap();
+    let authentication_key: Vec<u8> = if let Some(key) = authentication_key {
+        bcs::to_bytes(&key).unwrap()
+    } else {
+        vec![]
+    };
+    resource_account_create_resource_account(seed, authentication_key)
```

Comment:
My preference would be to just put this inline below. I don't think we'll see this being used across the codebase and there's a bit of nuance here that doesn't generalize -- the way the seed is being used and the notion of account address instead of authenticationkey.

Reply:
I have removed the function altogether and put the content in the caller.